pkg/reconciler/dependencybuild/dependencybuild.go

```diff
@@ -246,27 +248,34 @@ func (r *ReconcileDependencyBuild) handleStateAnalyzeBuild(ctx context.Context,
 			Invocations      [][]string
 			EnforceVersion   string
 			IgnoredArtifacts []string
+			ToolVersion      string
+			JavaVersion      string
 		}{}
 
 		if err := json.Unmarshal([]byte(buildInfo), &unmarshalled); err != nil {
 			r.eventRecorder.Eventf(&db, v1.EventTypeWarning, "InvalidJson", "Failed to unmarshal build info for AB %s/%s JSON: %s", db.Namespace, db.Name, buildInfo)
 			return reconcile.Result{}, err
 		}
-		//for now we are ignoring the tool versions
-		//and just using the supplied invocations
+		// for now we are ignoring the tool versions
+		// and just using the supplied invocations
 		buildRecipes := []*v1alpha1.BuildRecipe{}
-		_, maven := unmarshalled.Tools["maven"]
+		_, maven := unmarshalled.Tools["mvn"]
```

Comment:
can you clarify why this change is needed .... feels like it would break existing assumptions, no?

does this make sense to you @stuartwdouglas ?  ... maybe I am missing something elsewhere that necessitates this.

Reply:
It looks like this change was made by @stuartwdouglas in 0530131c. I wonder if we wanted to change "maven" to "mvn" to match the actual name of the binary? This would require changing the JSON annotation or field name in the Java code as well which generates this.